From 76508dcaa107fef1d72616d781c2d138a8641352 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wim.taymans@collabora.co.uk>
Date: Mon, 26 Mar 2012 15:59:01 +0200
Subject: [PATCH 03/40] v4l2src: don't error in shutdown

Don't log with the ERROR category when we are stopping because we are shutting
down.

Fixes: https://bugzilla.gnome.org/show_bug.cgi?id=672824
---
 sys/v4l2/gstv4l2src.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sys/v4l2/gstv4l2src.c b/sys/v4l2/gstv4l2src.c
index c0f362f..4c06aac 100644
--- a/sys/v4l2/gstv4l2src.c
+++ b/sys/v4l2/gstv4l2src.c
@@ -824,7 +824,8 @@ gst_v4l2src_fill (GstPushSrc * src, GstBuffer * buf)
   /* ERROR */
 error:
   {
-    GST_ERROR_OBJECT (src, "error processing buffer");
+    GST_DEBUG_OBJECT (src, "error processing buffer %d (%s)", ret,
+        gst_flow_get_name (ret));
     return ret;
   }
 }
-- 
1.8.3

